addons/crm/models/crm_team.py

```diff
@@ -441,12 +406,8 @@ def _allocate_leads(self, work_days=1):
               are already removed at return of this method;
           }, ...
         """
-        if work_days < 0.2 or work_days > 30:
-            raise ValueError(
-                _('Leads team allocation should be done for at least 0.2 or maximum 30 work days, not %.2f.', work_days)
-            )
 
-        BUNDLE_HOURS_DELAY = int(self.env['ir.config_parameter'].sudo().get_param('crm.assignment.delay', default=0))
+        BUNDLE_HOURS_DELAY = float(self.env['ir.config_parameter'].sudo().get_param('crm.assignment.delay', default=0))
```

Comment:
Docstring should also be updated to float then ^^

Reply:
Yeah probably I didn't check all the docstring super carefully